Products/PortalTransforms/Transform.py

```diff
@@ -176,6 +175,7 @@ def _load_transform(self):
         return transform
 
     security.declarePrivate('manage_beforeDelete')
+
```

Comment:
Although putting these lines directly above the function is not PEP8 compliant, I'm not a fan of putting a blank line in between because it increases the chance that someone will rename or move a function but forget to bring the security declaration along with it.

Ideally we should switch to using the decorator-based security declarations.